build-tools/entrypoint.builder.sh

```diff
@@ -7,7 +7,15 @@
 USER_ID=${LOCAL_USER_ID:-9001}
 
 echo "Starting with UID : $USER_ID"
-adduser -D -s /bin/bash -u $USER_ID user
 export HOME=/home/user
 
-exec /sbin/su-exec user "$@"
+if [ -v GOSU_VERSION ]; then
```

Comment:
`if [ -x /sbin/su-exec ]` would make the check more readable. Otherwise you need to look at the Dockerfile to see what this variable means